iOS/DuckDuckGo/SettingsViewModel.swift

```diff
@@ -803,6 +897,19 @@ extension SettingsViewModel: AutofillSettingsViewControllerDelegate {
     }
 }
 
+// MARK: DataImportViewControllerDelegate
+extension SettingsViewModel: DataImportViewControllerDelegate {
+    @MainActor
+    func dataImportViewControllerDidFinish(_ controller: DataImportViewController) {
+        AppDependencyProvider.shared.autofillLoginSession.startSession()
+        pushViewController(legacyViewProvider.loginSettings(delegate: self,
```

Comment:
Take users directly to the Passwords screen if they started the import flow from “Import Passwords” on the Settings screen